common/djangoapps/enrollment/api.py

```diff
@@ -250,7 +251,30 @@ def get_course_enrollment_details(course_id):
         }
 
     """
-    return _data_api().get_course_enrollment_info(course_id)
+    cache_key = "enrollment.course.details.{course_id}".format(course_id=course_id)
```

Comment:
(might be worth adding a test case to make sure the Django caching backend is okay with this -- I can't remember how it works with unicode off the top of my head)